test/integ/test_key_pair_authentication.py

```diff
@@ -64,6 +64,49 @@ def test_get_token_from_private_key(input_account, expected_account):
     )
 
 
+@pytest.mark.skipolddriver
+def test_regionless_url_JWT_token_validity(request, db_parameters):
+
+    test_user = "admin"
+
+    db_config = {
+        "account": "amoghorgurl-keypairauth_test_alias.testdns",
+        "user": "admin",
+        "role": "ACCOUNTADMIN",
+        "timezone": "UTC",
+    }
+
+    db_config_with_pw = {
+        "account": "amoghorgurl-keypairauth_test_alias.testdns",
+        "user": "admin",
+        "password": "Password1",
+        "role": "ACCOUNTADMIN",
+        "timezone": "UTC",
+    }
+
+    cnx = snowflake.connector.connect(**db_config_with_pw)
+    cursor = cnx.cursor()
+    cursor.execute(
+        """
+    use role accountadmin
+    """
+    )
+
+    private_key_der, public_key_der_encoded = generate_key_pair(2048)
+
+    cnx.cursor().execute(
+        """
+    alter user {user} set rsa_public_key='{public_key}'
+    """.format(
+            user=test_user, public_key=public_key_der_encoded
+        )
+    )
+
+    db_config["private_key"] = private_key_der
+    with snowflake.connector.connect(**db_config) as _:
```

Comment:
I believe that you don't actually have to name it anything if you don't want to use it
```suggestion
    with snowflake.connector.connect(**db_config):
```